src/agent.ts

```diff
@@ -0,0 +1,92 @@
+import { FindingType, FindingSeverity, Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+
+
+const CREATE_FUNCTION_ABI: string = "function createAgent(uint256 agentId, address owner, string metadata, uint256[] chainIds) public";
+const UPDATE_FUNCTION_ABI: string = "function updateAgent(uint256 agentId, address owner, string metadata, uint256[] chainIds) public";
```

Comment:
This ABI is incorrect. You could test the bot with a real transaction that updates a bot using `npm run tx txHash(i.e. 0x...)` to make sure that it's returning a finding.